src/uct/rocm/base/rocm_base.c

```diff
@@ -422,6 +424,10 @@ ucs_status_t uct_rocm_base_mem_query(uct_md_h md, const void *addr,
         mem_attr_p->dmabuf_offset = dmabuf_offset;
     }
 
+    if (mem_type == UCS_MEMORY_TYPE_ROCM) {
```

Comment:
what abount UCS_MEMORY_TYPE_ROCM_MANAGED?

Reply:
At the moment we are not doing anything special for ROCM_MANAGED memory, it is treated as host memory.